webpack.config.js

```diff
@@ -50,16 +58,25 @@ const config = {
 			},
 			{
 				test: /\.s?css$/,
-				use: [
-					{ loader: 'style-loader' },
-					{ loader: 'css-loader' },
-					{ loader: 'postcss-loader' },
-					{ loader: 'sass-loader' }
-				]
+				use: ExtractTextPlugin.extract( {
+					use: [
+						{ loader: 'raw-loader' },
+						{ loader: 'postcss-loader' },
+						{
+							loader: 'sass-loader',
+							query: {
+								outputStyle: isProduction ? 'compressed' : 'nested'
+							}
+						}
+					]
+				} )
 			}
 		]
 	},
 	plugins: [
+		new ExtractTextPlugin( {
+			filename: './[name]/build/style.css'
+		} ),
 		new webpack.DefinePlugin( {
```

Comment:
>I think we could drop this for now, since React is no longer a dep

Yeah, it might be nice to have in the future to change logic based on environment, but can probably drop it for now.